projects/managed-hubs/support.md

```diff
@@ -173,24 +174,62 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
+   
+   If the support steward does not have the capacity, they should ping the **Engineering Manager** (or the delegated person) so he can secure resources to resolve that issue on the fly (see step 6 below).
+   
+   The support steward **should not** work on issues with impact lower than `critical` (unless they are assigned as part of the "planned" reactive work in the context of a running sprint (see step 6 below).
+5. `Who: Support steward`
+
+   **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 5. Also, move the status of the ticket to the "Pending" state.
+   
+6. `Who: Support steward`
+
+   **Communicate status**. Once we have an issue created to track the next steps, send a message to the Community Representative letting them know about the situation: after some initial investigation and no immediate fix, a follow-up issue was created that will be assigned in the future accordingly to the current prioritization. Also, let them know what the next steps will be. Here's a template to help guide you:
 
    ```
    Hello { NAME }, thanks for sending us your request. We've concluded that
    this is not related to an outage or diminished service, so we'll incorporate
    this request into our team's work planning process. We'll send an update
    when we've got a plan for completing this request.
    ```
-5. **Prioritize the request**. Work with our team to decide how we should prioritize this request relative to the other work we need to do. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests).
-6. **Resolve the request**. Our team should work together to prioritize and resolve this request according to its impact.
-7.  **Confirm resolution**. Once we have resolved a support request, send a message to the Community Representative to confirm that we believe it is resolved. In FreshDesk, mark the incident as {guilabel}`Resolved`.
-8.  **Close the request**. If the Community Representative confirms that their request has been fulfilled, or if we have not gotten a response after 48 working hours, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
+7. `Who: Engineering Manager (currently assigning reactive work) or someone delegated by Engineering Manager if he is not available`
+
+   **Prioritize the request**. Any non-`critical` issue should wait to be included in our sprints (on Wednesdays, every other week) to be worked out as part of the "planned" reactive work. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests) to decide how we should prioritize this request relative to the other work we need to do. We should be fully transparent about the support queue to our Community Representatives if they ping us for updates.
```

Comment:
I don't a clear simple alternative strategy to propose, but this part is the biggest painpoint for me.

Imagine there is a support ticket with that I consider "High" priority that I think I could resolve in 90 minutes for example, then I'd stress out about delegating it to be resolved in the future:
- could cause problem as its "High" priority being considered first in up to two weeks time
- will cause additional overhead work for everyone involved

Those situations make me _very itchy_ to see it resolved directly instead.

Writing this up makes me think that a key issue with the procedure is that we:
- only distinguish non-critical from critical in our response, not low-high prio
- don't have a workflow that relate at all to the duration until the next time we allocate task, even though its probably relevant

I'm imagining a High prio request arriving after a sprint meeting, and the support steward isn't able to resolve it in 30 minutes but perhaps in 90 minutes, then its not comfortable to be a support steward.

Reply:
To me, the point is that sprint planning is the moment when we consider task importance + difficulty and turn that into priorities and a plan. If we encourage people to respond to support issues without going through the "zoom out, look at all of our work, now decide what is most important" process, I suspect that the support steward will feel that itchiness and bias themselves towards operational support toil, which is what we're trying to avoid here.

Another reason for this: we want to have mechanisms that encourage the team to share knowledge about operating our infrastructure. If engineers are reacting and resolving immediately, it will be harder to intentionally pair engineers together with the purpose of learning. My hope would be that in our sprint planning, we might say things like "support ticket X is probably going to touch Y part of our infrastructure, so let's pair engineers A and B so that B can see how this is resolved".

Reply:
> "zoom out, look at all of our work, now decide what is most important" 
I generally concur with @choldgraf.

Btw, the idea of having "critical" vs. "high-medium-low" tries to make the "critical" state, and hence, the immediate reaction as "high bar" as possible. If there is something "High" in priority that can be solved in 90 minutes, then it should be scheduled in the future (next sprint) OR, if we have **enough time** in the current sprint, and after completing the goal-oriented task, it could be scheduled on the fly as "planned reactive work" (accessing a curated list of "High" issues existing in the current sprint board: https://github.com/orgs/2i2c-org/projects/21/views/4). But again, that should be a low-probability scenario.

Additionally, this conversation also raised the point that we need a better process to put tasks in the planned reactive bucket for specific sprints. Hopefully, we can start planning that using the view I shared above.